Features/Addlanguage.feature

```diff
@@ -0,0 +1,57 @@
+﻿Feature: Addlanguage
+
+A short summary of the feature
+
+Scenario:1. Verify User able to Add New Language
+	Given User logs into Mars Portal 
+	When User creates a new Language record 'English' 
+	Then new Language 'English' should be successfully created 
+
+	Scenario:2. Verify User Gets Duplicate Error message 
+Given User logs into Mars Portal 
+When User enters new Language 'English' 
+Then User gets Error message as Duplicate Data
+
+	Scenario:3. Verify User able to Update New Language
+	Given User logs into Mars Portal 
+	When clean the existing data and add new language 'Malay' 
+	Then new Language 'Malay' Updated successfully
+
```

Comment:
Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

E.g of above scenarios
Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.

The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).

Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).